src/marqo/s2_inference/clip_utils.py

```diff
@@ -421,7 +420,7 @@ def __init__(self, model_type: str = "multilingual-clip/ViT-L/14", device: str =
                             truncate: bool = True, **kwargs) -> None:
 
         self.model_name = model_type
-        self.model_info = model_registry._get_multilingual_clip_properties()[self.model_name]
+        self.model_info = get_multilingual_clip_properties()[self.model_name]
```

Comment:
we can use `self.model_info = kwarge.get("model_properties", dict())` and remove the added model_registry information later.

Reply:
Let's move this into a separate refactor PR, to decouple these